content/en/blog/_posts/the-case-for-kubernetes-limits/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "The Case for Kubernetes Resource Limits: Predictability vs. Efficiency"
+date: TODO
+slug: the-case-for-kubernetes-resource-limits-predictability-vs-efficiency
+---
+
+**Author:** Milan Plžík
+
+
+
+_Milan Plzik is a Site Reliability Engineer on the [Grafana Labs](http://grafana.com) platform team, which maintains and improves internal infrastructure and tooling used by the product teams. As part of the infrastructure squad, he is primarily trying to make Kubernetes upgrades as smooth as possible. He likes going down the rabbit hole of various interesting Kubernetes issues._
+
+_Disclaimer: This article reflects the author's personal opinion; feel free to disagree. Also, there are a lot of subjective ideas that have not made it through any kind of rigorous testing. Use at your own risk._
+
+There’s been quite a lot of posts suggesting that not using Kubernetes resource limits might be a fairly useful thing (e.g., [here](https://home.robusta.dev/blog/stop-using-cpu-limits/) or [here](https://erickhun.com/posts/kubernetes-faster-services-no-cpu-limits/) ). The points made there are totally valid – it doesn’t make much sense to pay for compute power that will not be used due to limits, nor to artificially increase latency. This post strives to argue that limits have their legitimate use as well.
```

Comment:
Please avoid “here” in hyperlinks, even for blog articles. See https://kubernetes.io/docs/contribute/style/style-guide/ - much of which actually doesn't apply to blog writing.

Reply:
Fixed; replaced with actual article names.